core/src/main/java/io/kestra/core/models/tasks/retrys/AbstractRetry.java

```diff
@@ -58,4 +61,9 @@ public static <T> RetryPolicy<T> retryPolicy(AbstractRetry retry) {
         return new RetryPolicy<T>()
             .withMaxAttempts(1);
     }
+
+    public enum Behavior {
+        RETRY_FAILED,
+        NEW_EXECUTION
```

Comment:
```suggestion
        CREATE_NEW_EXECUTION
```
to align with the RETRY_X starting with a verb + it seems worth making it more verbose but explicit - WDYT?